messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/jms/ConnectionFactoryAttributes.java

```diff
@@ -214,6 +219,16 @@ interface Common {
                 .setAttributeParser(AttributeParser.STRING_LIST)
                 .setAttributeMarshaller(AttributeMarshaller.STRING_LIST)
                 .setDeprecated(ModelVersion.create(13, 0, 0), false)
+                .setAlternatives("deserialization-allow-list")
+                .setRestartAllServices()
```

Comment:
Did you try adding .setFlags(AttributeAccess.Flag.ALIAS)  to this and the other deprecated one?

That will suppress this from read-resource and read-attribute-group output unless the op includes the 'aliases=true' param.

Reply:
I did it when registering the attribute in the ResourceDefinition: https://github.com/wildfly/wildfly/pull/15603/files#diff-a3324164caaccf163c3d99b6c312a49217e3d66db679305d55ab59ac8a4884c6R87